components/consumers/pdf/main.go

```diff
@@ -63,18 +68,16 @@ func main() {
 		scanID = r[0].OriginalResults.ScanInfo.ScanUuid
 	}
 
-	cleanupRun := func(msg string, cleanup func() error) {
-		if err := cleanup(); err != nil {
-			slog.Error(msg, "error", err)
-		}
-	}
-
 	pw, err := playwright.NewClient()
 	if err != nil {
 		log.Fatalf("could not launch playwright: %s", err)
 	}
 
-	defer cleanupRun("could not stop Playwright: %w", pw.Stop)
+	defer func() {
+		if err := pw.Stop(); err != nil {
+			slog.Error("could not stop Playwright", "error", err)
```

Comment:
⛏️ to leverage the log field better you could do `slog.Error("could not stop Playwright", slog.String("err", err.Error())`